### PR TITLE
Fix DG appendix and add tips

### PR DESCRIPTION
**Pull request by CreditableX**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -321,7 +329,7 @@ Format: `find n/KEYWORD [MORE_KEYWORDS]`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
 * Only full words will be matched e.g. `Han` will not match `Hans`
 * Buyers matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang
```
Comment (Solomon0519) [inline docs/UserGuide.md:332]: I think you accidentally deleted a `

File: docs/UserGuide.md [hunk 2]
```diff
@@ -588,6 +596,10 @@ Clears all entries from the property list.
 
 Format: `clear`
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
```
Comment (NgZiXin) [inline docs/UserGuide.md:599]: For this mentioned at the top in quick start

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -842,14 +842,17 @@ Expected: Properties with addresses containing the keywords `Bishan`, `Marsiling
...
+* We plan to add support for special characters in names such as Lupita Nyong'o, Adib S/O Tharman etc. This can be done by updating the regex of our commands to accept certain special characters.
 
 ### Restriction of property types
-* Property types will be given restrictions in the future such that nonsensical types will not be accepted. We will limit types to common ones such as `HDB`, `Landed Property`, `Condominium` etc.
+* Property types will be given restrictions in the future such that nonsensical types will not be accepted. We will create an `enum` with common types such as `HDB`, `Landed Property`, `Condominium` etc.
 
 ### Improved duplication detection
 * Person duplication detection will be changed from the current method of matching names to checking for a repeated phone number OR email as these are more unique identifiers.
 
+### `Find` using more parameters
```
Comment (NgZiXin) [inline docs/DeveloperGuide.md:853]: Add a line to say this will be extended to meet-up, buyers and properties

File: docs/UserGuide.md [hunk 4]
```diff
@@ -305,6 +305,14 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [b/BUDGET] [t/TAG]…​`
 * You can remove all the buyer’s tags by typing `t/` without
     specifying any tags after it.
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
```
Comment (NgZiXin) [inline docs/UserGuide.md:308]: add this tip to delete instead

**Review by NgZiXin — COMMENTED** [review]
LGTM

**Pull request merged**